pkg/platform/packet/template.go

```diff
@@ -220,6 +226,17 @@ output "dns_entries" {
 }
 {{- end }}
 
+{{- if eq .Config.DNS.Provider "route53" }}
+provider "aws" {
+  version = "2.48.0"
+  alias   = "default"
```

Comment:
`alias` is not needed.

Reply:
I've re-added the provider as it was before #422. Also, other providers in this file do have `alias` so if we remove it we should do it for all of them.

Reply:
I think we should have some development guidelines for quality and technical debt vs inconsistency, as this is recurring topic. I'll let other decide on this PR then :)

Reply:
Good point. I've already started forming a list of proposed guidelines for this kind of cases. I'll add "consistency vs. quality".

Reply:
Looks like `alias` **is** needed since the upgrade path fails when not explicitly injecting the provider using its alias into the `dns` module. This likely has to do with the fact that the providers were configured this way *to begin with*, however if we care about not breaking upgrades from `v0.1.0` looks like we don't have a choice but keep the `alias` there.